crates/criteria_policy_base/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+edition = "2024"
+name    = "criteria_policy_base"
+version = "0.1.0"
+
+[dependencies]
+anyhow                = "1"
+const_format          = "0.2"
+kubewarden-policy-sdk = "0.14"
+serde                 = { version = "1.0", features = ["derive"] }
+serde_json            = "1.0"
+
+[features]
+annotations = []
+env_var     = []
+labels      = []
+
+[dev-dependencies]
+# this is a transitive dependency of kubewarden-policy-sdk, we don't
+# care about the specific version, we just have to enable the k8s' version feature
+k8s-openapi = { version = "0", default-features = false, features = ["v1_32"] }
+rstest      = "0.26.0"
+serde_yaml  = "0.9"
```

Comment:
As this policy is a workspace now, I think we could point to dependencies from the workspace file (the policy Cargo.tml file) when both subcrate and policy share them. Something like this:

```
rstest = {workspace=true}
```

Reply:
You're right. But as we discussed in other synced communication channel, we could move the policy itself to be a workspace member, instead of the root of the workspace. Therefore, we could define all the common dependencies in the workspace Cargo file and use it in all the workspace members. Including the dev dependencies. Something similar to what the [opentelemetry-rust does](https://github.com/open-telemetry/opentelemetry-rust/blob/86dfd4b3db00ecb910d718ed367c672d7c586121/Cargo.toml#L21)

However, this will required mode changes in the github-actions and CI. Which is something that we do not want to do now. Therefore, I'm marking this as resolved.